IOS/SwiftSeedProject/SwiftSeedProject/Infrastructure/PersistenceData.swift

```diff
@@ -20,54 +20,58 @@ class PersistenceData: Persistence {
     }
     
     public func getAll<T: PersistenceObject>(entityName: String) -> [T] {
-        let objectContext = coreDataManager.privateChildManagedObjectContext()
+        let objectContext = coreDataManager.mainManagedObjectContext
```

Comment:
we need to review this because on this new way you are using always the main thread which means that in some cases you are blocking this.